apps/cow-fi/app/(learn)/learn/topics/page.tsx

```diff
@@ -4,10 +4,11 @@ import { getArticles, getCategories } from '../../../../services/cms'
 import { Color } from '@cowprotocol/ui'
 import { TopicsPageComponent } from '@/components/TopicsPageComponent'
 
-export default async function Page() {
-  const categoriesResponse = await getCategories()
-  const articlesResponse = await getArticles()
+export default async function TopicsPage() {
+  const articlesResponse = await getArticles({ pageSize: 200 })
```

Comment:
Use the constant with page size?
Also, some places use 200, other 100. Should they all be the same?